internal/server/server_test.go

```diff
@@ -1,20 +1,43 @@
 package server
 
 import (
+	"log"
 	"net/http"
 	"net/http/httptest"
 	"os"
 	"testing"
 
+	"github.com/gorilla/sessions"
+	"github.com/kanopy-platform/k8s-auth-portal/pkg/mocks"
 	"github.com/stretchr/testify/assert"
 )
 
 var testHandler http.Handler
+var server *Server
```

Comment:
Should be able to simplify this by changing the server `New` return type to `*Server`

Reply:
Fixed